src/handlers/payout/index.ts

```diff
@@ -34,8 +35,17 @@ export const handleIssueClosed = async () => {
   console.log({ assignee });
   const recipient = await getWalletAddress(assignee.login);
   if (!recipient) {
-    log.info(`Recipient address is missing`);
-    await addCommentToIssue(`@${assignee.login} would you please post your wallet address here?`, issue.number);
+    const {
+      data: { state_reason },
+    } = await context.octokit.issues.get({
+      owner: payload.organization!.login,
+      repo: payload.repository.name,
+      issue_number: issue.number,
+    });
+    if (!RESERVED_USERNAMES[assignee.login] && state_reason === "completed") {
```

Comment:
`state_reason === "completed"`

If there's not predefined types for `completed`, we need to define them for issue status and use it here.

Reply:
what do u mean by that? `state_reason === "completed"` means that the issue was closed as completed (provided by the GitHub API), 
as oppossed to "not planned"

Reply:
What I mean by that is that we shouldn't use hardcoded values `completed` here. we can define types for `completed` and `not planned` somewhere and use it